src/activity_handlers.cpp

```diff
@@ -292,6 +293,8 @@ void activity_handlers::butcher_finish( player_activity *act, player *p )
             g->m.spawn_item( p->pos(), "bone", bones, 0, age );
             p->add_msg_if_player( m_good, _( "You harvest some usable bones!" ) );
         }
+    } else if( meat == "bone_tainted" ) {
```

Comment:
I was keeping the behavior the same as before. Before, failing to butcher a skeleton's bones would print a message about clumsy butchering, but failing to butcher a zombie's (or anything that can yield something other than bone) bones wouldn't. That behavior seems reasonable enough.